modules/battery_manager/ajax/change_active_status.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * Battery Manager status changer
+ *
+ * Used by Battery Manager module to activate or deactivate entries in Test Battery
+ *
+ * PHP Version 7
+ *
+ * @category Loris
+ * @package  Battery_Manager
+ * @author   Victoria Foing <victoria.foing@mcin.ca>
+ * @license  Loris license
+ * @link     https://www.github.com/aces/Loris
+ */
+
+// Determine whether entry should be activated or deactivated
+if (sanitize('action', 'get') !== null) {
+    $action = sanitize('action', 'get');
+    if ($action === "activate") {
+        echo changeStatus('Y');
```

Comment:
travis seems to be complaining about the use of `echo` here, seemingly due to the `@return` of the function doc of changeStatus, maybe change it to `@return string`?